apps/src/common.rs

```diff
@@ -438,8 +442,12 @@ impl Http09Conn {
 
 impl HttpConn for Http09Conn {
     fn send_requests(
-        &mut self, conn: &mut quiche::Connection, target_path: &Option<String>,
+        &mut self, conn: &mut quiche::Connection, client_args: &ClientArgs,
     ) {
+        let ClientArgs {
+            dump_response_path: target_path,
+            ..
+        } = client_args;
```

Comment:
This destructuring isn't helping much. Suggest you drop it and just access the fields where you need them e.g. client_args.dump_response_path